editor.planx.uk/src/@planx/components/Pay/Public/InviteToPayForm.tsx

```diff
@@ -148,7 +143,7 @@ const InviteToPayForm: React.FC<InviteToPayFormProps> = ({
         <ReactMarkdownOrHtml source={nomineeDescription} openLinksOnNewTab />
       </Typography>
       <StyledForm onSubmit={formik.handleSubmit}>
-        <InputLabel label="Full name (optional)" htmlFor="payeeName">
```

Comment:
This is required in the form and db so I've removed "(optional)" from the label. We don't currently use this field anywhere though so this is something we could make optional if we wanted to.